.scalafmt.conf

```diff
@@ -1,17 +1,12 @@
-version = 3.4.3
+version = 3.7.2
 runner.dialect = scala213
 preset = default
 align.preset = most
 maxColumn = 120
 project.git = true
+project.layout = standardConvention
 docstrings.style = AsteriskSpace
 align.tokens."+" = [
   {code = ":=", owner = "Term.ApplyInfix"}
 ]
 rewrite.rules = [RedundantBraces, RedundantParens]
-
-fileOverride {
```

Comment:
This is no longer needed because scalafmt now provides a `project.layout = standardConvention` which applies the correct Scala dialects according to sbt's folder layout as long as its the standard layout (which in this projects case it is).